### ISSUE TITLE
v 3+ (BUG/FEATURE UPDATE) Replace the name-based location look-up with lat/long for the 'Calculate Benefits' / iTree API call

### ISSUE THREAD

**Issue by TreeMama** [TreeMama/iSeaTree-React-Prototype#371]
ToDO: iTreeAPI has updated their spec to deprecate the name based location search.

The updated API documents are here (and the specific section on the location parameters is pg 13): [iTreeAPI Documentation](https://dtbe-api.daveyinstitute.com/static/home/documents/API-Usage-Information_The-Davey-Tree-Benefits-Engine.pdf)

We need to update our API call accordingly to use lat/long instead of the name based location search that @JonathanKoerber put in a few years ago (see: git[ #148](https://github.com/TreeMama/iSeaTree-React-Prototype/issues/148) & #74 ).

Long story short: We should use the lat / long in the api call (to iTreeAPI) BUT still save the location name to our Firebase storage (as it does now) AND in the Calculated Tree Benefits dialog show the users the name of the place.

Only thing changing here is the api call parameters. Keep everything else.

![api](https://user-images.githubusercontent.com/62637055/212385232-a053224b-5aec-4acb-b0a4-f6251005ba83.JPG)

**Comment by TreeMama** [comment]
verified on 4.1.5
